### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by SyXavier** [schreibfaul1/ESP32-audioI2S#93] [Q&A]
Hi, Wolle schreibfaul1, Can you explain the Audio::i2s_ mclk_ pin_select()?  I don't understand the method.

**Comment by schreibfaul1** [comment]
Most DACs do not require a master clock, MCLK is generated internally. If MCLK should be necessary in certain cases, this clock cannot be applied to any GPIO, only GPIO1 ... 3 are permitted.
